src/components/Navigation.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+const Navigation = () => (
+  <div className="nav-container">
+    <nav className="nav">
+      <h1>BookStore CMS</h1>
+      <ul className="nav-links">
+        <li>
+          <Link to="/">Book</Link>
+        </li>
+        <li>
+          <Link to="/categories">Category</Link>
```

Comment:
- [ ] There is no route named `/categories`, kindly consider renaming this to `/category` to match the existing route or rename the route in the `App` component to `/categories`

Reply:
- [ ] As pointed out by the previous reviewer, this link in your navigation gets you to `/categories`. However, checking your Router in your `App` component, there's no route with this name. The one listed there is named `/category`. Please modify one or the other, I think that `/categories` would fit best. 🤓 🔧